src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Link.js

```diff
@@ -28,6 +28,9 @@ export default class Link extends React.Component<FieldTypeProps<LinkValue>> {
                 types: {
                     value: unvalidatedTypes,
                 } = {},
+                excluded_types: {
+                    value: unvalidatedExcludedTypes,
+                } = {},
```

Comment:
Looks like a new feature which should be documented. Personally I would prefer that this is splitted into feature targetting 2.5 and bugfix targetting 2.4.